scripts/_build-and-test-all-tram-cdc.sh

```diff
@@ -2,73 +2,69 @@
 
 set -e
 
-if [ -z "$DOCKER_COMPOSE" ]; then
-    echo setting DOCKER_COMPOSE
-    export DOCKER_COMPOSE="docker-compose -f docker-compose-${DATABASE}.yml -f docker-compose-cdc-${DATABASE}-${MODE}.yml"
-else
-    echo using existing DOCKER_COMPOSE = $DOCKER_COMPOSE
-fi
-
 ./gradlew $GRADLE_OPTIONS $* :eventuate-cdc-service:clean :eventuate-cdc-service:assemble
 
-. ./scripts/set-env-${DATABASE}-${MODE}.sh
+. ./scripts/set-env.sh
+
+docker="./gradlew tram${DATABASE}${MODE}Compose"
 
-$DOCKER_COMPOSE stop
-$DOCKER_COMPOSE rm --force -v
+export COMPOSE_SERVICES=","
```

Comment:
Why COMPOSE_SERVICES=","? Looks strange

Reply:
It is not needed, I just missed it in this place.